src/test/java/core/basesyntax/MainAppTest.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainAppTest {
+    public static void main(String[] args) {
+        Machine[] machines = { new Truck(), new Bulldozer(), new Excavator() };
 
+        for (Machine machine : machines) {
+            machine.doWork();
```

Comment:
Ensure that the 'doWork' method in each Machine type (Truck, Bulldozer, Excavator) outputs an informative message specific to the machine type. For example, 'Truck started its work.'